src/app/scenarios/edit-scenario/edit-scenario.component.test.js

```diff
@@ -10,19 +13,50 @@ describe('EditScenarioComponent', () => {
     let routeParams;
     const id = 123;
 
+    class ScenarioServiceMock {
+        set reqStatForTest(status) {
+            this._status = status;
+        }
+        getScenario() { return new MockPromise(true, scenario); }
+        updateScenario() { return new MockPromise(true); }
+        delete(data) {
+            const status = this._status || 200;
+            return {
+                subscribe(fn) {
```

Comment:
this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix